src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ApplicationTest.kt

```diff
@@ -2797,6 +2797,9 @@ class ApplicationTest : IntegrationTestBase() {
           )
         }
 
+        // Create a clarification note that already has a domain event
+        createAssessmentClarificationNote(assessment, user, 3, true)
```

Comment:
if we used named args is the comment still required?

Reply:
I don't think it's worth worrying too much - this code will be deleted soon :)